x-pack/plugins/licensing/server/license.test.ts

```diff
@@ -4,10 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ILicense } from './types';
+import { ILicense } from '../common/types';
+import { LICENSE_STATUS } from '../common/constants';
+import { LicenseFeature } from '../common/license_feature';
```

Comment:
Should this file be moved to `../common`?

Reply:
We *could*, but it would complicate the test since the setup is different for the server vs. the client.